PurchasesTests/Purchasing/RCIntroEligibilityTests.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  RCIntroEligibilityTests.swift
+//  PurchasesTests
+//
+//  Created by Andrés Boedo on 7/1/20.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import XCTest
+import Nimble
+@testable import Purchases
```

Comment:
`@testable import` allows us to import `IntroElegibilityStatus`, an `internal` enum in Swift